src/api/OpenAIAPI.js

```diff
@@ -6,24 +6,23 @@ const openAI = new OpenAI({
 });
 
 async function generateRecipe(ingredients) {
-  try {
-    const response = await openAI.chat.completions.create({
-      messages: [{ role: "user", content: getContent(ingredients) }],
-      model: "gpt-3.5-turbo",
-    });
+  const response = await openAI.chat.completions.create({
+    messages: [{ role: "user", content: getContent(ingredients) }],
+    model: "gpt-3.5-turbo",
+  });
 
-    if (!response.choices.length)
-      return { error: 'There was an generating the recipe. Please try again.' }
+  if (!response.choices.length)
+    throw new Error('There was an generating error the recipe. Please try again.');
 
-    const recipeObject = JSON.parse(response.choices[0].message.content);
-    return recipeObject;
-  } catch (error) {
-    return { error: `There was an error generating the recipe. Please try again. ${error}`};
-  }
+  const recipeObject = JSON.parse(response.choices[0].message.content);
+  if (!recipeObject.name || !recipeObject.ingredients || !recipeObject.instructions || !recipeObject.cookTime)
+    throw new Error('There was an error generating the recipe. Please try again.');
+
+  return recipeObject;
 }
 
 function getContent(ingredients) {
-  return `I'm building a recipe finder app where users input ingredients, and I provide them with a recipe they can make using those ingredients. Please provide a recipe in JSON format with the following keys: name (string), ingredients (string[]), instructions (string[]), and cookTime (string). Ingredients: ${ingredients}`;
+  return `I'm building a recipe finder app where users input ingredients, and I provide them with a recipe they can make using those ingredients. Please provide a recipe in JSON format with the following keys: name (string), ingredients (string[]), instructions (string[]), and cookTime (string). Do not put numbers in the instructions. Ingredients: ${ingredients}`;
```

Comment:
Have you tried asking ChatGPT to improve this prompt? You may be able to make this prompt a little stronger.